.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - run: npm ci
```

Comment:
Oh that's interesting, didn't knew about `npm ci` 👍 
Do you think the `build:pre` action should be updated in `package.json` too?

Reply:
Yeah maybe if you want to make sure that your `npm install` isn't updating your `package-lock.json` in CI (which it seems to be when I ran `build:pre` on my machine). 

But maybe it might be worth looking into [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces) or [lerna](https://github.com/lerna/lerna) as it seems like there are multiple packages in this repo and it may be cleaner to manage it that way.